src/selectors/selectors.js

```diff
@@ -43,9 +44,9 @@ export const useRootSelector = <T>(selector: (state: RootReducerState) => T): T
 // Most commonly used selectors
 export const useFiatCurrency = () => useRootSelector(fiatCurrencySelector);
 export const useRatesPerChain = () => useRootSelector(ratesPerChainSelector);
-export const useChainRates = (chain: Chain) => useRatesPerChain()[chain] ?? {};
+export const useChainRates = (chain: Chain) => useRatesPerChain()[chain] ?? EMPTY_OBJECT;
```

Comment:
Uhhh, not sure on `EMPTY_OBJECT` and `EMPTY_ARRAY`, looks too much hassle for nothing 👀

Reply:
Original code would return new object instance (`{}`) each time the `useRatesPerChain()[chain]` returned null/undefined. Hence if any `useEffect`/`useMemo`/etc relied on that value then, it would be triggered on each render because the identity of the object changed. This would not be a problem if `useRatesPerChain` used `useMemo`, but I believe that returning the same instance of `EMPTY_OBJECT` is simpler solution.

Reply:
Maybe we can then rename `EMPTY_OBJECT` to `LOCKED_ EMPTY_OBJECT` or `FROZEN_ EMPTY_OBJECT`, wdyt?